data/test/biometric_datasource_impl_test.dart

```diff
@@ -41,48 +41,192 @@ import 'fixture/mock/mock_fixtures.dart';
 
 void main() {
   isAvailable();
+  getAvailableBiometric();
+  getBiometricSetting();
+  saveBiometricSetting();
+  authenticate();
 }
 
 void isAvailable() {
   group('biometric_data_source_impl isAvailable test', () {
     BiometricDataSourceImpl _biometricDataSourceImpl;
-    MockLocalAuthenticationService _localAuthenticationService;
+    MockLocalBiometricService _localBiometricService;
     MockBiometricExceptionThrower _biometricExceptionThrower;
     SharedPreferences _sharedPreferences;
 
     Future _initDataSource() async {
-      SharedPreferences.setMockInitialValues({
-        Constant.biometricSettingState: BiometricState.disabled
-      });
-
+      SharedPreferences.setMockInitialValues({});
       _sharedPreferences = await SharedPreferences.getInstance();
-      _localAuthenticationService = MockLocalAuthenticationService();
+      _localBiometricService = MockLocalBiometricService();
       _biometricExceptionThrower = MockBiometricExceptionThrower();
-      _biometricDataSourceImpl = BiometricDataSourceImpl(_localAuthenticationService, _biometricExceptionThrower, _sharedPreferences);
+      _biometricDataSourceImpl = BiometricDataSourceImpl(_localBiometricService, _biometricExceptionThrower, _sharedPreferences);
     }
 
     test('isAvailable should return success with valid data', () async {
       await _initDataSource();
 
-      when(_localAuthenticationService.isAvailable()).thenAnswer((_) async => true);
+      when(_localBiometricService.isAvailable()).thenAnswer((_) async => true);
+      await _initDataSource();
```

Comment:
There are 2 `_initDataSource` there

Reply:
It coincided when I rebase code. I will delete it.